aspnet/fundamentals/logging.rst

```diff
@@ -16,23 +16,46 @@ Implementing Logging in your Application
 
 Adding logging to a component in your application is done by requesting either an ``ILoggerFactory`` or an ``ILogger<T>`` via :doc:`dependency-injection`. If an ``ILoggerFactory`` is requested, a logger must be created using its ``CreateLogger`` method. The following example shows how to do this within the ``Configure`` method in the ``Startup`` class:
 
-.. literalinclude:: logging/sample/src/LoggingSample/Startup.cs
+.. literalinclude:: logging/sample/src/TodoApi/Startup.cs
 	:language: c#
 	:linenos:
-	:lines: 17-27
+	:lines: 17-28
 	:dedent: 8
 	:emphasize-lines: 2,8-9
 
-When a logger is created, a category name or source must be provided. By convention this string is hierarchical, with categories separated by dot (``.``) characters. Some logging providers have filtering support that leverages this convention, making it easier to locate logging output of interest. In the above example, the logging is configured to use the built-in `ConsoleLogger <https://github.com/aspnet/Logging/blob/1.0.0-beta6/src/Microsoft.Framework.Logging.Console/ConsoleLogger.cs>`_ (see `Configuring Logging in your Application`_ below). To see the console logger in action, run the sample application using the ``web`` command, and make a request to configured URL (``localhost:5000``). You should see output similar to the following:
+When a logger is created, a category name or source must be provided. By convention this string is hierarchical, with categories separated by dot (``.``) characters. Some logging providers have filtering support that leverages this convention, making it easier to locate logging output of interest. In the above example, the logging is configured to use the built-in `ConsoleLogger <https://github.com/aspnet/Logging/blob/1.0.0-beta7/src/Microsoft.Framework.Logging.Console/ConsoleLogger.cs>`_ (see `Configuring Logging in your Application`_ below). To see the console logger in action, run the sample application using the ``web`` command, and make a request to configured URL (``localhost:5000``). You should see output similar to the following:
```

Comment:
> When a logger is created, a category name or source must be provided

When I first read this I thought I had to choose between providing a category name or a source, as if they are two separate concepts. In the API it's called a category name, so let's just say that and then define what it is in the next sentence. Something like: "When a logger is created, a category name must be provided. The category name specifies the source of the logging events"